### DISCUSSION TITLE
[@types/meteor] Accounts methods missing parameters

### DISCUSSION THREAD

**Discussion by KeithGillette** [DefinitelyTyped/DefinitelyTyped#55173] [Issues with a @types package] [answered]
Per the Meteor Accounts documentation, the typings for [Accounts.sendEnrollmentEmail](https://docs.meteor.com/api/passwords.html#Accounts-sendEnrollmentEmail), [Accounts.sendResetPasswordEmail](https://docs.meteor.com/api/passwords.html#Accounts-sendResetPasswordEmail),  [Accounts.sendVerificationEmail](https://docs.meteor.com/api/passwords.html#Accounts-sendVerificationEmail) as well as [Accounts.findUserByUsername](https://docs.meteor.com/api/passwords.html#Accounts-findUserByUsername), and [Accounts.findUserByEmail](https://docs.meteor.com/api/passwords.html#Accounts-findUserByEmail) are missing optional function parameters.

[a comment by an automated account is not shown]

**Comment by ToastHawaii** [comment, marked as answer]
Fixed with 86497e2d0023bbcab8edc24760be31610597ab9e
